client/src/components/CKANPresentedOptionsField.scss

```diff
@@ -50,4 +50,11 @@
   .form__field-description {
     font-style: italic;
   }
+
+  // Override read-only styling
+  .readonly .form__field-holder & {
+    background-color: #f7f8fa;
```

Comment:
Any chance of a variable here? You've used $body-bg further down in a similar block

Reply:
🤦‍♂️